source/blog/2019-01-06-ovirt-openshift-part-1.html.md

```diff
@@ -0,0 +1,148 @@
+---
+title: oVirt and OKD
+author: rgolan
+tags: oVirt, oVirt 4.2, open source virtualization,kubernetes, openshift, external-storage, flexdriver, provisioner
+date: 2019-01-06 09:01:00 UTC
+---
+
+<img src="/images/blog/2019-01-06/boxhead.png" width="400px"/>
+
+
+
+This is a series of posts to demonstrate how to  install  OKD 3.11 on oVirt and what you can do with it.
+
+**Part I**   -  How to intall OKD 3.11 on oVirt
+
+
+
+# How to intall OKD 3.11 on ovirt (4.2 and up)
+
+Installing OKD on oVirt has many advantages, and its also gotten a lot easier these days. Admins and users who want to take container platform management for a spin, on oVirt, will be encouraged by this.
```

Comment:
Installation of Kubernetes on top of oVirt has a couple of advantages:
-Virtualizing the control plane for Kubernetes - provide HA/backup/affinity capabilities to the controllers and allowing hardware maintenance cycles
-providing persistent volume for containers via the IAAS, without the need for additional storage array dedicated to Kubernetes
-Allowing a quick method to build up/tear down Kubernetes clusters, providing hard tenency model via VMs between clusters

Reply:
done